code/modules/reagents/dispenser/dispenser_presets.dm

```diff
@@ -89,7 +89,10 @@
 			/obj/item/reagent_containers/chem_disp_cartridge/tonic,
 			/obj/item/reagent_containers/chem_disp_cartridge/sodawater,
 			/obj/item/reagent_containers/chem_disp_cartridge/lemon_lime,
+			/obj/item/reagent_containers/chem_disp_cartridge/sugar,
+			/obj/item/reagent_containers/chem_disp_cartridge/watermelon,
 			/obj/item/reagent_containers/chem_disp_cartridge/clean_kois,
+			/obj/item/reagent_containers/chem_disp_cartridge/banana,
```

Comment:
```suggestion
			/obj/item/reagent_containers/chem_disp_cartridge/clean_kois,
```

Add a bag of sugar to the bartender's backrooms instead. Add a carton of banana and watermelon juice to the booze-o-mat instead.

Reply:
Done